src/Autosuggest.js

```diff
@@ -215,6 +249,8 @@ export default class Autosuggest extends Component {
       focusedSectionIndex: sectionIndex,
       focusedSuggestionIndex: suggestionIndex
     });
+
+    this.onSuggestionFocused(sectionIndex, suggestionIndex);
```

Comment:
Shouldn't we call `this.onSuggestionUnfocused()` instead here? If so, how come tests didn't catch this?